package/MDAnalysis/analysis/rdf.py

```diff
@@ -55,6 +59,15 @@
 the first solvation shell :math:`N(r_1)` where :math:`r_1` is the position of
 the first minimum in :math:`g(r)`.
 
+In :class:`InterRDF_s`, we provide an option `density`. When `density` is
+``False``, it will return the RDF :math:`g_{ab}(r)`; when `density` is
+``True``, it will return the density of particle :math:`b` in a shell at
+distance :math:`r` around a :math:`a` particle, which is
```

Comment:
Note that `density=False` now means "calculate g(r)" and `density=True` means "calculate density".

Reply:
So we still need another PR to change the default `density`, right?

Reply:
No, this PR switched the behavior.

Reply:
See comment above https://github.com/MDAnalysis/mdanalysis/pull/2812#discussion_r453102797